pkg_resources/__init__.py

```diff
@@ -2221,12 +2221,21 @@ def _normalize_cached(filename, _cache={}):
         return result
 
 
+def _is_egg_path(path):
```

Comment:
> Could eventually be a one liner ?

It's just a renaming of the `_is_unpacked_egg()'. Yes it's a one-liner, but I think it saves enough typing (specially in an editor with completion), that I think it's worth it.